mzLib/Readers/ExternalResults/BaseClasses/ResultFile.cs

```diff
@@ -31,7 +43,7 @@ public List<TResult> Results
         protected ResultFile(string filePath, Software software = Software.Unspecified)
         {
             FilePath = filePath;
-            _results = new List<TResult>();
+            _results = null;
```

Comment:
I don't understand the rationale behind this change

Reply:
see above